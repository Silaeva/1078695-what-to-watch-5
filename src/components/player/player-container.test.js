import React from "react";
import renderer from "react-test-renderer";
import PlayerContainer from "./player-container";

jest.mock(`../player/player`, () => `Player`);

it(`Should PlayerContainer render correctly`, () => {
  const tree = renderer
      .create(
          <PlayerContainer currentFilmId={1} />)
      .toJSON();

  expect(tree).toMatchSnapshot();
});
