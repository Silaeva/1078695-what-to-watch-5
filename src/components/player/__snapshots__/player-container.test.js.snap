// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PlayerContainer render correctly 1`] = `
<Player
  currentFilmId={1}
  duration={0}
  isPlaying={true}
  onFullscreenClick={[Function]}
  onPlayBtnClick={[Function]}
  progress={0}
  renderPlayer={[Function]}
/>
`;
